sql/gen-sql-markdown.py

```diff
@@ -15,12 +15,18 @@
 # limitations under the License.
 #
 
-import sys
 import os
+import re
+import sys
 from collections import namedtuple
+from textwrap import dedent
+
+from mkdocs.structure.pages import markdown
 
 ExpressionInfo = namedtuple(
     "ExpressionInfo", "className name usage arguments examples note since deprecated")
+SQLConfEntry = namedtuple(
+    "SQLConfEntry", ["name", "default", "docstring"])
```

Comment:
I would rename `docstring` to something like `documentation` or `description`. docstring is a python term I believe :-).